molgenis-data/src/main/java/org/molgenis/data/jobs/Job.java

```diff
@@ -0,0 +1,182 @@
+package org.molgenis.data.jobs;
```

Comment:
Owh! Now I see where the "old" Job entity went, it got renamed to JobMetadata? And now the DTO is once again called Job?